File: src/main/java/com/technews/aggregate/subscribe/dto/SubscriberMailContents.java
package com.technews.aggregate.subscribe.dto;

import com.technews.aggregate.posts.domain.Post;
import com.technews.aggregate.releases.domain.Release;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.util.List;

@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class SubscriberMailContents {
    private List<Release> springReleases;
    private List<Release> javaReleases;

    private List<Post> springPosts;
    private List<Post> javaPosts;
}

File: src/main/kotlin/com/technews/aggregate/subscribe/dto/SubscriberMailContents.kt
package com.technews.aggregate.subscribe.dto

import com.technews.aggregate.posts.domain.Post
import com.technews.aggregate.releases.domain.Release

data class SubscriberMailContents(
    val springReleases: List<Release> = emptyList(),
    val javaReleases: List<Release> = emptyList(),
    val springPosts: List<Post> = emptyList(),
    val javaPosts: List<Post> = emptyList(),
)